src/central-server/admin-service/int-test/src/intTest/java/org/niis/xroad/cs/test/glue/SecurityServerApiStepDefs.java

```diff
@@ -198,4 +204,62 @@ private String randomSecurityServerId() {
                 randomAlphabetic(3), randomAlphabetic(3));
     }
 
+    @Step("user requests security servers list sorted by {string} {string}")
+    public void userRequestsSecurityServersListSortedBy(String sortBy, String order) {
+        final ResponseEntity<PagedSecurityServersDto> response = securityServersApi
+                .findSecurityServers(null, getPagingSortingParameter(sortBy, order));
+
+        validate(response)
+                .assertion(equalsStatusCodeAssertion(OK))
+                .execute();
+
+        savedResponse = response;
+    }
+
+    private PagingSortingParametersDto getPagingSortingParameter(String sortBy, String order) {
+        final PagingSortingParametersDto dto = new PagingSortingParametersDto();
+        dto.setSort(sortBy);
+        dto.setDesc("desc".equalsIgnoreCase(order));
+        return dto;
+    }
+
+    @Step("the list is sorted by {string} {string}")
+    public void theListIsSortedByServerCodeDesc(String fieldExpression, String order) {
```

Comment:
step sounds really generic, maybe it should be under separate definition file?

Reply:
Moved to CommonStepDefs